src/rules/interfaceNameRule.ts

```diff
@@ -15,9 +15,11 @@
  * limitations under the License.
  */
 
+import * as utils from "tsutils";
 import * as ts from "typescript";
 
 import * as Lint from "../index";
+import { isUpperCase } from "./variableNameRule";
```

Comment:
I'd recommend moving this to a separate file that is imported by both this rule and variableNameRule. Helps avoid circular dependencies and in a few edge cases, circular dependencies can cause runtime issues to pop up which is never fun. Also, no semantic reason for `isUpperCase` to live in `variableNameRule` imo

(sorry for the drive-by comment after this was merged! Just happened to notice it haha)